base/src/main/java/io/spine/validate/StringFieldValidator.java

```diff
@@ -101,4 +101,9 @@ protected boolean isNotSet(String value) {
         boolean result = value.isEmpty();
         return result;
     }
+
+    @Override
+    protected boolean requiredAllowed() {
+        return true;
```

Comment:
All the overloads of this method return a constant. Please consider passing the value in a `super(...)` ctor call.